clients/client-topia/src/controllers/User.ts

```diff
@@ -38,30 +50,37 @@ export class User extends SDKController {
    * @summary
    * Returns all assets owned by User when an email address is provided.
    */
-  async fetchAssetsByEmail(ownerEmail: string): Promise<object | ResponseType> {
+  async fetchAssets(): Promise<void | ResponseType> {
     try {
-      const response: AxiosResponse = await this.topiaPublicApi().get(
-        `/assets/my-assets?email=${ownerEmail}`,
-        this.requestOptions,
-      );
-      return response.data;
+      const response: AxiosResponse = await this.topiaPublicApi().get(`/assets/my-assets`, this.requestOptions);
+      const tempAssetsMap: { [key: string]: Asset } = {};
+      for (const i in response.data) {
+        const assetDetails = response.data[i];
+        tempAssetsMap[assetDetails.id] = new Asset(this.topia, assetDetails.id, {
+          attributes: assetDetails,
+        });
+      }
+      this.#assetsMap = tempAssetsMap;
     } catch (error) {
       throw this.errorHandler({ error });
     }
   }
 
   /**
    * @summary
-   * Returns all scenes owned by User when an email address is provided.
+   * Returns all scenes owned by User
    */
-  async fetchScenesByEmail(): Promise<object | ResponseType> {
+  async fetchScenes(): Promise<void | ResponseType> {
     try {
-      if (!this.email) throw this.errorHandler({ error: new Error("There is no email associated with this user.") }); // throw a new Error so the stack trace goes to errorHandler
-      const response: AxiosResponse = await this.topiaPublicApi().get(
-        `/scenes/my-scenes?email=${this.email}`,
-        this.requestOptions,
-      );
-      return response.data;
+      const response: AxiosResponse = await this.topiaPublicApi().get("/scenes/my-scenes", this.requestOptions);
```

Comment:
changes to public api need to be made to make this work.

right now `my-scenes` endpoint will return the scenes owned by the owner of the interactive key pair. It needs to get the scenes owned by the user identified by the session

Reply:
should probably have a lengthy discussion about this as it will have huge implications on security, etc.

Reply:
Why would my-scenes return another User's scenes? Seems like if that is the direction we want to go then we should deprecate this in favor of an enhance getUserScenes endpoint. Does this need to block this PR given the only thing I changed was remove the unused email param and map results to class attribute instead of returning. This isn't new and is relatively unchanged in this PR.

Reply:
The PR I submitted today on public API is meant to address this.

The question is whether we want to serve the scenes of the user who interacted with the asset or the scenes of the owner of the interactive key.

Maybe we close these endpoints to interactive key pairs? If 3rd party devs have access to scene ids of other users, is that problematic?

It's probably best to talk about this. it's pretty nuanced. Hard to write it all out